specifyweb/frontend/js_src/lib/components/QueryBuilder/FieldFilter.tsx

```diff
@@ -560,6 +565,24 @@ export const queryFieldFilters: RR<
     types: ['checkbox'],
     hasParser: true,
   },
+  name: {
```

Comment:
`name` is a quite generic term.
is this filter specifically for geo trees?
then perhaps call it `geoName` or something like that?
similar for `age` and `range`